mod/ckeditor/views/default/elgg/ckeditor/config.js

```diff
@@ -8,8 +8,8 @@ define(function(require) {
 		removeButtons: 'Subscript,Superscript', // To have Underline back
 		allowedContent: true,
 		baseHref: elgg.get_site_url(),
-		removePlugins: 'liststyle,contextmenu,tabletools',
-		extraPlugins: 'blockimagepaste,autogrow',
```

Comment:
`blockimagepaste `is not a plugin in the plugin folder, same for `contextmenu`.  `liststyle `and `tabletools `were already being removed.

#9172